dev/com.ibm.ws.app.manager/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -81,3 +81,6 @@ appmgmt.startAfter.desc=Specifies applications that are required to start before
 
 classloader.name=Classloader
 classloader.desc=Defines the settings for an application classloader.
+
+appmgmt.targetDir=Target Expansion Directory
+appmgmt.targetDir.desc=Allows the user to set the location where the autoExpanded application resides. The default location is ${server.config.dir}/apps/expanded/.
```

Comment:
Change,
"Allows the user to set the location where the autoExpanded application resides. The default location is ${server.config.dir}/apps/expanded/."
to,
"The targetDir attribute allows a user to set the location where the autoExpanded application is located. The default location is ${server.config.dir}/apps/expanded/."

Reply:
done.